_data/companies.yml

```
@@ -1684,3 +1684,10 @@
   visitors: Restricted
   events: Restricted
   last_update: 2020-03-12
+  
+- name: Motorola Solutions
+  wfh: Strongly encouraged where possible, until at least 2020-04-06
```

```suggestion
  wfh: Strongly encouraged where possible until at least 2020-04-06
```

That comma isn't really needed. Are you ok with dropping it?

---

I'm not bothered either way.